AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/ReportBase.java

```diff
@@ -23,11 +27,17 @@ public void setReportId(int reportId) {
         this.reportId = reportId;
     }
 
+
     @Column(name = "status", nullable = false)
     public byte getStatus() {
-        return status;
+        if (status <= 100) {
+            return status;
+        } else {
+            return Byte.parseByte("INVALID");
+        }
```

Comment:
So what happens if a user manages to submit an invalid status code? Would they be allowed to do so?

It looks like they're allowed to submit things with invalid status codes and it's just that when we retrieve them we say they're invalid!

Reply:
Sorry I didn't catch this the first time around!

Reply:
I changed the backend file to return an error message